### PR TITLE
Documentation updates 

### PR DESCRIPTION
**Pull request by alanking**
Addresses #174 
Addresses #216 
Addresses #232

Plus, a tweak to the test Compose project for using 4.3.2.

### CODE REVIEW

File: docker/ingest-test/README.md [hunk 1]
```diff
@@ -4,16 +4,15 @@
...
+When testing against an alternative version of iRODS, there are three variables in the `docker-compose.yml` file which must be changed prior to the build step. For example, if testing against iRODS 4.3.2:
 ```
   irods-catalog-provider:
     build:
       args:
-          irods_version: 4.2.11-1~bionic
-          irods_version_major_minor: 4.2
+          irods_version: 4.3.2-0~jammy
+          irods_version_major_minor: 4.3
           py_version: ""
```
Comment (trel) [inline docker/ingest-test/README.md:16]: this needs to be a 3?

File: README.md [hunk 2]
```diff
@@ -165,7 +159,7 @@ Note that the UnicodeEncodeError type of remapping is unconditional, whereas the
 an event handler's character_map method being defined.  Also, if a UnicodeEncodeError-style ingest is performed on a
 given object, this precludes character mapping being done for the object.
 
-### operation mode ###
+### operation mode
```
Comment (korydraughn) [inline README.md:162]: operation -> Operation?

**Review by korydraughn — COMMENTED** [review]
Looks good.

File: README.md (hunk 2 above)
Comment (alanking) [reply README.md:162]: Sure. I'm going to move this whole section up closer to the other event handler section(s) too

Comment (alanking) [reply README.md]: Done

File: docker/ingest-test/README.md (hunk 1 above)
Comment (alanking) [reply docker/ingest-test/README.md]: Updated and confirmed that tests passed with this configuration.

**Comment by korydraughn** [comment]
All review comments have been resolved. If there's nothing left to do for this, squash it.

**Comment by alanking** [comment]
Squashed

**Review by korydraughn — APPROVED** [review]
Pound/Merge it if you're happy with it.

**Comment by alanking** [comment]
Addresses #229 

Forgot to link in commit https://github.com/irods/irods_capability_automated_ingest/commit/d3c95b6d9652bf5274b52dab8f008e06ca67288b

**Pull request merged**